core/src/tpu.rs

```diff
@@ -103,6 +106,25 @@ impl Tpu {
             poh_recorder,
             tpu_coalesce_ms,
         );
+
+        let (weighted_packet_sender, weighted_packet_receiver) = unbounded();
+
+        let transaction_weight_stage = FindPacketSenderStakeStage::new(
```

Comment:
nit rename to `_find_packet_sender_stake_stage`

Reply:
+1